src/getHumanAge.test.js

```diff
@@ -7,4 +7,52 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should have right length', () => {
+    const result = getHumanAge(2, 3);
+
+    expect(result).toHaveLength(2);
+  });
+
+  test('should return an Array', () => {
+    const result = getHumanAge;
+
+    expect(Array.isArray(result));
+  });
+
+  test('should have expected values', () => {
+    const result = getHumanAge(0, 0);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should have expected values bv1', () => {
+    const result = getHumanAge(5, 5);
+
+    expect(result).toEqual([0, 0]);
+  });
```

Comment:
The test description 'should have expected values bv1' is repeated multiple times. Consider using unique descriptions for each test case to avoid confusion and to better document what each test is verifying.